src/lib/testPaymentExpiry.js

```diff
@@ -125,13 +145,21 @@ TransferTester.prototype.loadCaseExpiry = function * (caseID) {
     json: true
   })
   if (caseRes.statusCode !== 200) {
-    throw new UnacceptableExpiryError('Unexpected remote error: ' + caseRes.statusCode + ' ' + caseRes.body)
+    throw new IlpError({
+      code: 'T00',
```

Comment:
Seems like a remote error should be treated differently. Shouldn't there be a way to say "I tried but the next guy didn't work"?